docs/advisors/checks/postgresql-unused-index-check.md

```diff
@@ -0,0 +1,26 @@
+# Check for relations with unused indexes for PostgreSQL
+
+
+## Description
+
+This check will scan all databases in the cluster and list relations with indexes that have not been used since the last statistics reset.
```

Comment:
```suggestion
This check scans all databases in the cluster and lists relations with indexes that have not been used since the last statistics reset.
```